spin/command.go

```diff
@@ -14,7 +14,8 @@ import (
 // Run provides a shell script interface for the spinner bubble.
 // https://github.com/charmbracelet/bubbles/spinner
 func (o Options) Run() error {
-	isTTY := term.IsTerminal(os.Stdout.Fd())
+	isOutTTTY := term.IsTerminal(os.Stdout.Fd())
```

Comment:
should this be `isOutTTY` or is the `TTTY` intentional?

Reply:
Looks great :)

![demo](https://github.com/user-attachments/assets/8eb7fcff-bd55-46e9-945e-1869fbc3dba2)